projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/query-management/components/query-check-your-answers/query-check-your-answers.component.html

```diff
@@ -19,7 +21,22 @@ <h1 class="govuk-heading-l">{{ 'Review query response details' | rpxTranslate }}
     <dl class="govuk-summary-list govuk-!-margin-bottom-0">
       <div class="govuk-summary-list__row">
         <dt class="govuk-summary-list__key">
-          {{ 'Response message' | rpxTranslate }}
+          {{ 'Full name' | rpxTranslate }}
+        </dt>
+        <dd class="govuk-summary-list__value">
+          {{ formGroup.controls['fullName'].value }}
+        </dd>
+        <dd class="govuk-summary-list__actions">
+          <a class="govuk-link" (click)="goBack()">
+            {{ 'Change' | rpxTranslate }}
+          </a>
+        </dd>
+      </div>
+    </dl>
+    <dl class="govuk-summary-list govuk-!-margin-bottom-0">
+      <div class="govuk-summary-list__row">
+        <dt class="govuk-summary-list__key">
+          {{ context === 'Responded' ? 'Response message' : 'Query body' | rpxTranslate }}
```

Comment:
@DavidJayakumar - similar to @MateiHMCTS's comment, please use the enum value instead of the string `'Responded'`